plugins/flytekit-sqlalchemy/Dockerfile.py3.10

```diff
@@ -0,0 +1,28 @@
+FROM python:3.10-slim-buster
+
+WORKDIR /app
+ENV VENV /opt/venv
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV PYTHONPATH /app
+
+RUN apt-get update -y
+RUN apt-get install git -y
+
+RUN pip install awscli
+RUN pip install gsutil
+
+ARG VERSION
+
+# Virtual environment
+RUN python3.8 -m venv ${VENV}
```

Comment:
these should match the versions from the base image?

Reply:
good catch @eapolinario !